src/Microsoft.Diagnostics.EventFlow.Inputs.EventFlowTelemetryProcessor/Microsoft.Diagnostics.EventFlow.Inputs.EventFlowTelemetryProcessor.csproj

```diff
@@ -0,0 +1,38 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
so this project file produces the NuGet package? It can replace those three files? https://github.com/Microsoft/ApplicationInsights-dotnet/tree/develop/src/Core/NuGet/Core